client/blocks/login/login-form.jsx

```diff
@@ -1089,13 +1094,14 @@ export class LoginForm extends Component {
 							{ ! isBlazePro && <p className="login__form-terms">{ socialToS }</p> }
 							{ shouldRenderForgotPasswordLink && this.renderLostPasswordLink() }
 							<div className="login__form-action">
-								<FormsButton
```

Comment:
[FormsButton](https://github.com/Automattic/wp-calypso/blob/fffc4d2b1a4f4e0f7c6574419234b1e450838689/client/components/forms/form-button/index.jsx#L9) is basically an A8C submit Button with some default text, which isn't being used here.